bpi_features/bpi_features.features.user_permission.inc

```diff
@@ -1,90 +0,0 @@
-<?php
```

Comment:
This feature has never actually been used, i.e. it was never included in the feature module.
It was a mistake from the beginning, that this file was in the git repo.